scripts/travis/upload-to-registry.sh

```diff
@@ -85,7 +85,7 @@ upload_images() {
         docker push ${registry}/${image}:latest
         ;;
       *)
-        docker push ${registry}/${image}
+        docker push --all-tags ${registry}/${image}
```

Comment:
Is it a good opportunity to move this out of "travis"?

Reply:
Sure @jpkrohling let me update the PR removing all the travis references and remove files which are no longer needed.